lib/game_teams.rb

```diff
@@ -0,0 +1,31 @@
+class GameTeams
+  attr_reader :game_id,
+              :team_id,
+              :HoA,
+              :result,
+              :settled_in,
+              :head_coach,
+              :goals,
+              :shots,
+              :tackles,
+              :pim,
+              :powerPlayOpportunities,
+              :powerPlayGoals,
+              :faceOffWinPercentage,
+              :giveaways,
+              :takeaways
+  def initialize(row)
+    row.each do |k, v|
+      instance_variable_set("@#{k}" , v)
+    end
+    @faceOffWinPercentage = @faceOffWinPercentage.to_f
```

Comment:
Should we go back later and refactor the instance variables from pascalCase to snake_case?